dapper/dapperlib/types.go

```diff
@@ -493,7 +493,7 @@ func ParseRecords(in []Record, tAggr TimeAggrLevel) map[string]Table {
 }
 
 func contains(in []string, key string) bool {
-	if in == nil || len(in) == 0 {
+	if len(in) == 0 {
```

Comment:
could the `nil ` value be intended?

Reply:
This change is picked by golangci-lint, says it's unnecessary.

example:
https://go.dev/tour/moretypes/12